pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -1,24 +1,36 @@
-{ lib, buildPackages, fetchFromGitHub, fetchpatch, perl, buildLinux, ... } @ args:
+{ lib
+, fetchpatch
+, kernel
+, date ? "2021-05-20"
+, commit ? "ac91176cb1f48e218d19026977ebe52078297c32"
+, diffHash ? "1flkvq7v35ay6x9z9519bylah8l1a787s95gq6bn30izwzq1vx6p"
+, kernelPatches ? [] # will always be supplied by callPackage because it is also a top-level attribute
+, argsOverride ? {}
+, ...
+} @ args:
 
-buildLinux (args // {
-  version = "5.9.0-2020.11.20";
-  modDirVersion = "5.9.0";
+kernel.override ( args // {
 
-  src = fetchFromGitHub {
-    owner = "koverstreet";
-    repo = "bcachefs";
-    # commit does not exist on any branch on the target repository
-    rev = "6a505b63ed3003faf5000f19fd08bbd477d93fbc";
-    sha256 = "1rf34gzv9npafp1c3i6lymk3b0gnqp4rb0wl33pw6yrpgnsry3cc";
-  };
+  argsOverride = {
+    version = "${kernel.version}-${date}";
+    extraMeta = {
+      branch = "master";
+      hydraPlatforms = []; # Should the testing kernels ever be built on Hydra?
```

Comment:
```suggestion
```
If hydra can build them sure, why not?

Reply:
I think the idea behind this originally was to keep Hydra's workload low. I haven't seen anything to indicate that that rationale has changed.

Reply:
As long as the normal testing kernel isn't built on Hydra this one definitely shouldn't be, so I'll refrain from this for now. If you disagree, please open another PR changing it for both and someone who actually knows about what's acceptable to load Hydra with can help discuss it then.

Reply:
I don't think hydra will not notice a dent if we build this.